spec/ruby/core/main/using_spec.rb

```diff
@@ -129,4 +129,18 @@ def bar; 'quux'; end
 
     x.call_bar(cls2.new).should == 'bar'
   end
+
+  it "raises error when called from method in wrapped script" do
+    -> do
+      load File.expand_path('../fixtures/using_in_method.rb', __FILE__), true
+    end.should raise_error(RuntimeError)
+  end
```

Comment:
I added this test based on discussion [here](https://bugs.ruby-lang.org/issues/18960#note-5). cc @shugo